src/Menu/Builder.php

```diff
@@ -126,6 +126,8 @@ protected function addItem($itemKey, $direction, ...$newItems)
                 Arr::set($this->menu, $arrayPath, $menuItems);
```

Comment:
Change this line to this 
```php
Arr::set($this->menu, $arrayPath, $this->applyFilters($menuItems));
```

and remove the new added line below.

This should work and fix the TravisCI Error.

Reply:
Fixed :) Thanks for the review and the help!

Reply:
No problem 😄 I totally forgot to check this